package.json

```diff
@@ -39,7 +39,7 @@
     "pub": "npm run compile && npm run build && npm publish"
   },
   "peerDependencies": {
-    "g2": "^1.2.6"
+    "g2": "^2.0.0"
```

Comment:
peerDependencies with `g2@^1.2.6` [DO NOT](https://travis-ci.org/antvis/g2-react/jobs/201384258#L200) pass testing on node4.x